test/specs/core.helpers.tests.js

```diff
@@ -742,6 +742,41 @@ describe('Core helper tests', function() {
 		expect(canvas.style.width).toBe('400px');
 	});
 
+	it ('Should get padding of parent as number (pixels) when defined as percent (returns incorrectly in IE11)', function() {
```

Comment:
Ideally, we would need a unit test that doesn't check internals but creates a chart inside a container with percent padding and verify the chart dimensions (something similar to [this test](/chartjs/Chart.js/blob/master/test/specs/core.controller.tests.js#L278))

Reply:
I've removed call to now private function calculatePadding. 
I attempted to write test with chart but require a parent DIV to the wrapper so I can set the width as the padding percentage is taken from this and not from the wrapper.

Reply:
```javascript
it('**********Should calculate padding of parent container in pixels if as percent (IE11)', function(done) {
	var chart = acquireChart({
		type: 'pie',
		options: {
			responsive: true,
			maintainAspectRatio: true
		}
	}, {
		canvas: {
			style: 'width: 300px; height: 300px; position: relative'
		},
		wrapper: {
			style: 'padding: 10%; width: 300px; height: 300px;'
		}
	});

	expect(chart).toBeChartOfSize({
		dw: 240, dh: 240,
		rw: 240, rh: 240,
	});
});
```